src/_typography.scss

```diff
@@ -38,6 +38,33 @@
   }
 }
 
+@mixin link {
```

Comment:
This looks like asomething we can reuse it alot! great!
what do you think about to define more vars?

```
$c-link
$c-link-hover
$fw-link
$fw-link-hover
$td-link
$td-link-hover
[...]
```